index.bs

```diff
@@ -400,13 +443,21 @@ agent MUST run the following steps:
 1. Let |transport| have a
    <dfn attribute for="WebTransport">\[[Ready]]</dfn> internal
    slot, initialized to a promise.
+1. Let |transport| have an
+   <dfn attribute for="WebTransport">\[[OutgoingDatagrams]]</dfn> internal slot
+   initialized to the result of [=WritableStream/creating=] a {{WritableStream}},
+   its [=WritableStream/create/writeAlgorithm=] set to [=writeDatagrams=] given
+   |transport| as a parameter.
+1. Let |transport| have an
+   <dfn attribute for="WebTransport">\[[IncomingDatagrams]]</dfn> internal
+   slot initialized to the result of <a dfn for="ReadableStream">creating</a> a
+   {{ReadableStream}}. `transport.[[IncomingDatagrams]]` is provided datagrams
```

Comment:
How about ...to the result of creating a ReadableStream, its pullAlgorithm set to readDatagrams given _transport_ as parameter?

Reply:
Because it's a push-based reader,  not pull-based.

Reply:
Ah, sorry I misunderstood.